.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: c
+sudo: required
+compiler:
+  - gcc
+os: linux
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install -y dpkg-dev debhelper dkms lintian
+- sudo apt-get install linux-headers-`uname -r`
+script:
+- make
+# Getting PACKAGE_VERSION release
+- PACKAGE_VERSION=$(cat dkms.conf | grep  -i  PACKAGE_VERSION= | cut -d '"' -f 2)
+# Testing add, build, install and remove dkms
+- sudo git checkout-index --all --prefix=/usr/src/digimend-$PACKAGE_VERSION/
+- sudo dkms add -m digimend -v $PACKAGE_VERSION
+- sudo dkms build -m digimend -v $PACKAGE_VERSION
+- sudo dkms install -m digimend -v $PACKAGE_VERSION
+- sudo dkms uninstall -m digimend -v $PACKAGE_VERSION
+# Testing Debian package installation, configuration, and uninstall
+- dpkg-buildpackage -b -uc
+- lintian -i -I --profile debian --show-overrides ../digimend-dkms_*_`dpkg --print-architecture`.changes
```

Comment:
Could you please add a comment explaining what this line does? Thank you.

Reply:
Also, I think we can simply say `../digimend-dkms_*.changes` here. I don't think we would build more than one architecture, and even if we did, it would be nice to check all of them.